### PR TITLE
Margin Calculator and Pip calculator

### PR DESCRIPTION
**Pull request by arome-binary**
Changes:
- Update on the margin calculator
-  Creation of Pip calculator

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ x ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/trader-tools/pip-calculator/index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import Patterns from 'images/common/trade-tools/trade-tools-bg.png'
+
+const Hero = styled(Flex)`
+    height: 40rem;
+    background: var(--color-black);
+    background-image: url(${Patterns});
+    background-size: cover;
+`
+
+const MarginCalc = () => {
```
Comment (kevinw-binary) [inline src/pages/trader-tools/pip-calculator/index.js:17]: `const PipCalc`

File: src/pages/trader-tools/pip-calculator/index.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+                        {localize('Pip Calculator')}
+                    </Header>
+                </Container>
+            </Hero>
+            <PipCalculator />
+        </Layout>
+    )
+}
+
+export default WithIntl()(MarginCalc)
```
Comment (kevinw-binary) [inline src/pages/trader-tools/pip-calculator/index.js:33]: export default WithIntl()(PipCalc)

**Comment by kevinw-binary** [comment]
Since `PipCalculator` have some new text, it's better to push the request to `crowdin` instead of `master`

**Comment by sean-binary** [comment]
Closing this as this is no longer valid now 🙏

**Pull request closed without merging**